py/torch_tensorrt/dynamo/debug/_Debugger.py

```diff
@@ -18,13 +26,43 @@
 class Debugger:
     def __init__(
         self,
-        log_level: str,
+        log_level: str = "debug",
         capture_fx_graph_before: Optional[List[str]] = None,
         capture_fx_graph_after: Optional[List[str]] = None,
         save_engine_profile: bool = False,
-        logging_dir: Optional[str] = None,
+        profile_format: str = "perfetto",
+        engine_builder_monitor: bool = True,
+        logging_dir: str = tempfile.gettempdir(),
```

Comment:
I think tempdir is just `/tmp` or whatever, so we should do something like `/tmp/torch_tensorrt/debug` or something similar.